src/tests/HDFFile.cxx

```diff
@@ -257,22 +197,103 @@ class T_CommandHandler : public testing::Test {
       j.SetObject();
       Value nexus_structure;
       nexus_structure.SetObject();
-      Value streams;
-      streams.SetArray();
+
+      Value children;
+      children.SetArray();
+
+      {
```

Comment:
This function is long and reuses variable names from upper scope in several places, could really do with refactoring into smaller functions.